x-pack/plugins/canvas/server/lib/feature_check.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const isSecurityEnabled = server => {
+  const securityPlugin = server.plugins.security;
+  const xpackInfo = server.plugins.xpack_main.info;
+
+  return (
+    securityPlugin &&
+    xpackInfo.isAvailable() &&
+    xpackInfo.feature('security').isEnabled() &&
+    !xpackInfo.license.isOneOf('basic')
```

Comment:
Curious, is this really required? Won't `xpackInfo.feature('security').isEnabled()` be false if the license is wrong?

Reply:
So if you run elasticsearch with a basic license and 'xpack.security.enabled: true`, `server.xpack_main.feature('security').isEnabled()` will be true, but `server.xpack_main.feature('security').isAvailable()` will be false. Instead of checking the license, I can check `isAvailable()`, but I do need both checks in case you have a non-basic license but have security disabled.